mongodump/mongodump.go

```diff
@@ -38,22 +38,24 @@ type MongoDump struct {
 	InputOptions  *InputOptions
 	OutputOptions *OutputOptions
 
+	SkipUsersAndRoles bool
```

Comment:
What do you need this value if OutputOptions already has a value indicating whether to dump db users and roles...